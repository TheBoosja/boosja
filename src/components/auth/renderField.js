import React, { Fragment } from 'react';
import { Input, Error } from '../styled';

export default function renderField({ input, name, label, type, autoFocus, meta: { touched, error } }) {
	return (
		<Fragment>
			<Input {...input} name={name} type={type} placeholder={label} autoFocus={autoFocus} />
			<Error>{touched && error}&nbsp;</Error>
		</Fragment>
	);
}