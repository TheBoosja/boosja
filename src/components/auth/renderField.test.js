import renderField from './renderField';

describe('renderField', () => {
	const field = renderField({ input: {}, meta: {} });

	it('renders properly', () => {
		expect(field).toMatchSnapshot();
	});
});