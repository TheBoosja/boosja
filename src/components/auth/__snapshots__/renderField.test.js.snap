// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderField renders properly 1`] = `
<Unknown>
  <Input
    autoFocus={undefined}
    name={undefined}
    placeholder={undefined}
    type={undefined}
  />
  <Error>
    Â 
  </Error>
</Unknown>
`;
